Change document to Group Note and explain status of document - closes #432

benfrancis opened the pull request:
This PR changes the status of the document from ED (Editor's Draft) to NOTE (Group Note) and adds an explanation about the status of the document. Closes #432.

@ashimura I'm not sure whether the usual process is to land this change on the main branch first, or to do it when copying the specification to the publication folder, so I'm happy to change this PR to do the latter if necessary. In the meantime I would like to get a review of the proposed content itself.


<!--
    This comment and the below content is programmatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/benfrancis/wot-profile/pull/438.html" title="Last updated on Jul 30, 2025, 10:18 AM UTC (f655ac2)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-profile/438/b0a9374...benfrancis:f655ac2.html" title="Last updated on Jul 30, 2025, 10:18 AM UTC (f655ac2)">Diff</a>

mlagally reviewed (changes requested):
This PR and proposed text needs deeper review and has not yet been discussed in a call.

This wording will render the group note mostly useless, since it will scare off any potential implementer.
Until Profile 2.0 will be published (which may be far in the future), the Profile 1.0 is the only reasonable way to achieve interoperability between WOT implementations, unless customers adopt solutions from a single vendor, who has verified the interoperability without publicly visible interoperability guarantees.

**This reduces the potential market adoption of WoT.**

benfrancis commented:
Hi @mlagally, thank you for your feedback.

I agree that (whilst we need to accurately reflect the formal status of the document), WoT Profiles 1.0 is still essential as the only reasonable way to achieve out-of-the-box interoperability with WoT 1.x.

I have re-worded the Status of this Document section to attempt to address your concerns. I've tried to stick to objective facts and wording taken directly from the [W3C Process Document](https://www.w3.org/policies/process/20231103/#note-track) and [Types of documents W3C publishes](https://www.w3.org/standards/types/) document.

Please can you let me know whether this addresses your concerns?

benfrancis commented:
Profile call 2025-07-30:
- Ege says don't need to change ED to NOTE, usual process is to generate static HTML from ReSpec and copy the spec to the publications folder, making the document status change there
- Luca - Mention that there are existing implementations of this specification
- Kaz - OK with this direction, note mlagally's concerns

Ben will re-word and re-create PR when we move the document to the publication folder

benfrancis commented:
As discussed in the Profiles call today, we will not land this PR in its current state.

Let's continue the discussion of the content to add to the Status of this Document section in #432, and @ashimura will create a follow-up PR to add that note when copying a static version of the HTML (as a Group Note) to the publications folder.
